src/Compilers/CSharp/Test/Emit2/Semantics/ParamsCollectionTests.cs

```diff
@@ -4558,9 +4558,7 @@ static void Test(params System.Span<int> a)
 
             CompileAndVerify(
                 comp,
-                verify: ExecutionConditionUtil.IsMonoOrCoreClr ?
-                            Verification.FailsILVerify with { ILVerifyMessage = "[InlineArrayAsSpan]: Return type is ByRef, TypedReference, ArgHandle, or ArgIterator. { Offset = 0xc }" }
-                            : Verification.Skipped,
+                verify: Verification.Skipped,
```

Comment:
Curious now. Lets say I have the following code: 

```csharp
M(1);
M(2);
void M(params Span<int> i) { } 
```

After this change the compiler will optimize the calling code to roughly be: 

```
int tmp = 1;
M(new Span<int>(ref temp));
```

But do we create 1 or 2 temps here? Guessing we only create one today but want to check. This could be a significant optimization opportunity for us in the future.